src/mina-signer/src/rosetta.ts

```diff
@@ -20,6 +20,14 @@ function signatureFromHex(signatureHex: string): Signature {
   };
 }
 
```

Comment:
my particular use case is that i will have a payload from `client.signTransaction`. this it's expected that a user would call `Signature.fromJSON(signature)` first and pass that to the method below